src/Sulu/Bundle/PageBundle/Resources/js/views/Form/toolbarActions/TemplateToolbarAction.js

```diff
@@ -26,6 +27,22 @@ export default class TemplateToolbarAction extends AbstractFormToolbarAction {
             this.resourceFormStore.setType(this.defaultTemplate);
         }
 
+        if (this.router.attributes.parentId && this.webspace) {
+            ResourceRequester.get('pages', {
+                id: this.router.attributes.parentId,
```

Comment:
Extract that into an own variable like e.g. `formTypes` above, and reuse it in the surrounding `if` as well.

Reply:
Corrected